aiven/client/client.py

```diff
@@ -2836,6 +2838,48 @@ def byoc_permissions_set(
             body={"accounts": accounts, "projects": projects},
         )
 
+    def list_byoc_tags(self, organization_id: str, byoc_id: str) -> Mapping:
+        output = self.byoc_update(
+            organization_id=organization_id,
+            byoc_id=byoc_id,
+            # Putting all arguments to `None` makes `byoc_update()` behave like a `GET BYOC BY ID` API which does not exist.
+            deployment_model=None,
+            cloud_provider=None,
+            cloud_region=None,
+            reserved_cidr=None,
+            display_name=None,
+            tags=None,
+        )
+        return {"tags": output.get("custom_cloud_environment", {}).get("tags", {})}
+
+    def update_byoc_tags(self, organization_id: str, byoc_id: str, tag_updates: Mapping[str, str | None]) -> Mapping:
+        self.byoc_update(
+            organization_id=organization_id,
+            byoc_id=byoc_id,
+            deployment_model=None,
+            cloud_provider=None,
+            cloud_region=None,
+            reserved_cidr=None,
+            display_name=None,
+            tags=tag_updates,
```

Comment:
maybe i am missing something, but i don't see the `tags` argument defined in `byoc_update` method:
https://github.com/aiven/aiven-client/blob/9bc64ad5dc630690df9e000fa56250367b049bd0/aiven/client/client.py#L2727-L2737

Reply:
can you please add tests for the new methods in `client.py`? the tests below in `test_cli.py` are mocking these methods.

Reply:
> maybe i am missing something, but i don't see the `tags` argument defined in `byoc_update` method:

Yet they are in this diff.
https://github.com/aiven/aiven-client/blob/afeb11e546ec67261bb88815debbd03a32d9364d/aiven/client/client.py#L2737

Reply:
weird i couldn't see them